Latex/inhalt/Anhang.tex

```diff
@@ -10,7 +10,7 @@
 \renewcommand{\section}[1]{
 \par\refstepcounter{section}
 \sectionmark{#1}
-\addcontentsline{atoc}{section}{\protect\numberline{\thesection}#1}
+\addcontentsline{atoc}{section}{\bfseries\protect\numberline{\thesection}{\mdseries#1}}
```

Comment:
@DSczyrba was ändert das? Ich versteh's nicht ganz, eigentlich sollte irgendwas im anhang fett werden (?), ich seh' aber nur die fetteren "Anhang x" texte im verzeichnis - und das kommt ja durch die andere zeile, oder? ^^

Reply:
Du hast recht. Die Änderung macht nichts... Die scheint ausversehen mit rein geraten zu sein...
Werde ich morgen nochmal ändern.

Reply:
Nein stop. Die zeile hier macht das...
Die andere nicht. :D

Reply:
noch witziger: macht die andere zeile (auch in ihrer alten form) überhaupt _irgendwas_ ? wenn ich die entferne sieht irgendwie nichts anders aus 🤔